helm/values.yaml

```diff
@@ -61,6 +61,9 @@ microservice-chart:
     APPLICATIONINSIGHTS_PREVIEW_PROFILER_ENABLED: "false"
     ENABLE_AUDIT_APPENDER: "TRUE"
 
+  externalConfigMapValues:
```

Comment:
why externalConfigMapValues? do we have a config map defined in the infrastructure code?